packages/utils/src/types/TypeClass.ts

```diff
@@ -6,10 +6,22 @@ import {DataTypes} from '@subql/x-sequelize';
 type ValueOf<T> = T[keyof T];
 export type SequelizeTypes = string | ValueOf<typeof DataTypes>;
 
-export class TypeClass {
+export interface TypeToHashData {
```

Comment:
It would be nice to tie this to the type definitions. https://github.com/subquery/subql/tree/main/packages/utils/src/types/supported

If we add a new type in supported, e.g. BigFloat then this type no longer matches, there should be a compiler error but i don't think that would be the case